devices.js

```diff
@@ -1150,6 +1153,18 @@ const devices = [
         fromZigbee: [fz.CC2530ROUTER_state, fz.CC2530ROUTER_meta, fz.ignore_onoff_change],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['ptvo.switch'],
+        model: 'ptvo.switch',
+        vendor: 'Custom devices (DiY)',
+        description: '[Multi-channel relay switch](https://ptvo.info/zigbee-switch-configurable-firmware-router-199/)',
+        supports: 'hold, single, double and triple click, on/off',
+        fromZigbee: [fz.ptvo_switch_state, fz.ptvo_switch_buttons, fz.ignore_onoff_change, fz.ignore_multistate_change],
+        toZigbee: [tz.on_off, tz.ptvo_switch_trigger],
+        ep: (device) => {
+            return {'bottom_left': 1, 'bottom_right': 2, 'top_left': 3, 'top_right': 4, 'center_left': 5};
```

Comment:
There is a center left but no center right, shouldn't this just be center?

Reply:
If it is possible, then I'll change the button name. I think I tried this, but Z2M didn't accept commands from the MQTT side.

Reply:
It should as center is in this list; https://github.com/Koenkk/zigbee2mqtt/blob/dev/lib/util/utils.js#L31, can you post the log when it's not working?